[2024-03-01T16:34:21.638+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T10:52:00+00:00 [queued]>
[2024-03-01T16:34:21.652+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T10:52:00+00:00 [queued]>
[2024-03-01T16:34:21.652+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-01T16:34:21.896+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_csvs_from_urls> on 2024-03-01 10:52:00+00:00
[2024-03-01T16:34:21.918+0530] {standard_task_runner.py:60} INFO - Started process 22620 to run task
[2024-03-01T16:34:21.945+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'datapipeline_Take4', 'fetch_csvs_from_urls', 'scheduled__2024-03-01T10:52:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/datapipeline_best(2).py', '--cfg-path', '/tmp/tmpguyagc7g']
[2024-03-01T16:34:21.953+0530] {standard_task_runner.py:88} INFO - Job 119: Subtask fetch_csvs_from_urls
[2024-03-01T16:34:22.351+0530] {task_command.py:423} INFO - Running <TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T10:52:00+00:00 [running]> on host arun-VM
[2024-03-01T16:34:22.683+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='datapipeline_Take4' AIRFLOW_CTX_TASK_ID='fetch_csvs_from_urls' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T10:52:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T10:52:00+00:00'
[2024-03-01T16:34:23.901+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-03-01T16:34:23.907+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=datapipeline_Take4, task_id=fetch_csvs_from_urls, execution_date=20240301T105200, start_date=20240301T110421, end_date=20240301T110423
[2024-03-01T16:34:24.072+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T16:34:24.143+0530] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
